packages/schema/src/merge-schemas.ts

```diff
@@ -30,7 +30,8 @@ export function mergeSchemas(config: MergeSchemasConfig) {
     extractedResolvers.push(getResolversFromSchema(schema));
```

Comment:
Instead of the change above, we can basically change this `push` to`unshift`

Reply:
Hi @ardatan 

I have done some testing on this and using `extractedResolvers.unshift(getResolversFromSchema(schema))` still overrides the `resolvers` with those in the schema. 

I expected as in #4367 that the `resolvers` I provide to this package will override those that are in the `schema` with the same name. 

The current behavior that I am seeing is if a resolver in the `schema` and a `resolver` passed into this function have the same name neither ends up working.

What is your expected behavior? Which should have precedence the `schema` or the `resolver`?

Reply:
We'd love to accept a new PR :)

Reply:
Thanks @ardatan - I have created #4455